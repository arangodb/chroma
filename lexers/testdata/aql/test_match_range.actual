FOR c IN Characters
    FILTER c.age >= 13
    RETURN c.name