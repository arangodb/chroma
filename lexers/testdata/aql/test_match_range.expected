[
  {"type":"KeywordReserved","value":"FOR"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Text","value":" "},
  {"type":"KeywordReserved","value":"IN"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Characters"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"FILTER"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"age"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"\u003e="},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberFloat","value":"13"},
  {"type":"Text","value":"\n    "},
  {"type":"KeywordReserved","value":"RETURN"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"c"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"name"}
]
